.npmrc

```diff
@@ -1 +0,0 @@
-//registry.npmjs.org/:_authToken=${NPM_TOKEN}
```

Comment:
Why?

Reply:
Not really a part of it, mistake from my part.